doc/api/repl.markdown

```diff
@@ -106,6 +108,14 @@ the following values:
 You can use your own `eval` function if it has following signature:
 
     function eval(cmd, context, filename, callback) {
+      var result;
+      try {
+        result = vm.runInThisContext(cmd);
+      } catch (e) {
+        if (isRecoverableError(e)) {
```

Comment:
No, I can add it - it was only an example, but the actual implementation of `isRecoverableError` could be much complicated. Is there a psuedo-function I should show here?